library/src/main/java/com/okta/oidc/clients/web/WebAuthClientFactory.java

```diff
@@ -33,13 +33,15 @@
 public class WebAuthClientFactory implements ClientFactory<WebAuthClient> {
     private Executor mCallbackExecutor;
     private CustomTabOptions mCustomTabOptions;
+    private boolean mIsResultFragmentEnabled;
     private String[] mSupportedBrowser;
 
     public WebAuthClientFactory(@Nullable Executor callbackExecutor,
                                 CustomTabOptions customTabOptions,
+                                boolean isResultFragmentEnabled,
```

Comment:
This seems like we're making the developer choose if they want correct behavior, or broken behavior. Should we just remove the old code path instead?

Reply:
You have a point, thanks, I missed something here, current behavior, which works fine with native android clients, will be left untouched, but i forgot to add an option to https://github.com/okta/okta-oidc-android/blob/e7517b06060e57967411e6ee1d4dde4fb55dcbbc/library/src/main/java/com/okta/oidc/Okta.java#L45 to the builder with corresponding java doc explaining why would we want to use this option. ReactNative library is a wrapper around this sdk so we will just make proper configuration in the SdkBridge